src/unix/thread.c

```diff
@@ -302,6 +307,65 @@ int uv_sem_trywait(uv_sem_t* sem) {
   return -EINVAL;  /* Satisfy the compiler. */
 }
 
+#elif defined(__MVS__)
+
+int uv_sem_init(uv_sem_t* sem, unsigned int value) {
+  uv_sem_t a = semget(IPC_PRIVATE, 1, S_IRUSR | S_IWUSR);
```

Comment:
Can you separate the variable declarations from their assignments to be consistent with the preferred project style.